src/GetPdoValueHashStringCapableTrait.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace RebelCode\Storage\Resource\Pdo;
+
+use Dhii\Util\String\StringableInterface as Stringable;
+use InvalidArgumentException;
+
+trait GetPdoValueHashStringCapableTrait
+{
+    /**
+     * Hashes a query value for use in PDO queries when parameter binding.
+     *
+     * @since [*next-version*]
+     *
+     * @param int|float|bool|string|Stringable $value The value to hash.
+     *
+     * @return string The string hash.
+     */
+    protected function _getPdoValueHashString($value)
+    {
+        return ':' . hash('crc32b', $this->_normalizeString($value));
```

Comment:
Info: support for the `crc32b` hashing algorithm was added in PHP 5.1.2, and is continued through to 7.2.2, which is the current latest stable.